CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## 2.6.0 - 2018-05-16
+### Added
+- `YotiProfile`, which contains new `YotiAttribute` attributes, which you can get the sources and verifiers with .GetSources() and .GetVerifiers()
```

Comment:
`YotiProfile`, which contains `YotiAttribute` instances. `Yoti Attribute` allows you to get its sources and verifiers by means of the  `GetSources` and `GetVerifiers` methods.